docs/_templates/landing-page.html

```diff
@@ -0,0 +1,75 @@
+{%- extends "layout.html" %}
+
+{%- block css %}
+  {{ super() }}
+    <link href="_static/index.css" rel="stylesheet">
+{%- endblock css %}
+
+{%- block htmltitle %}
+    <title>Starlite Framework</title>
+{%- endblock %}
+
+{% block docs_main %}
+
+    <div id="landing-header">
+        <img src="_static/logo-banner-inline-dark.svg" class="only-dark">
+        <img src="_static/logo-banner-inline-light.svg" class="only-light">
+        <span id="landing-tagline">Effortlessly build performant APIs</span>
+    </div>
+
+    <div id="landing-overview">
+        <section>
+            <h2>Fully featured</h2>
+            <p>
+                Everything that's needed to build modern APIs, from data serialization and validation
+                to websockets, ORM integration, session management, authentication and more
+            </p>
+        </section>
+
+        <section>
+            <h2>Fast</h2>
+            <p>
+                Starlite puts great emphasis on developer experience and performance: It's one of the fastest ASGI
+                frameworks and developing with it is just as fast
+            </p>
+        </section>
+
+        <section>
+            <h2>(a)synchronous</h2>
+            <p>
+                Fully compliant with the ASGI specification, Starlite is asynchronous at heart, but synchronous
+                execution is not a second class citizen: Synchronous applications run without performance penalties
+            </p>
+        </section>
+
+        <section>
+            <h2>Opinionated yet flexible</h2>
+            <p>
+                Strongly encourages best practices, but tries not to get in the way of adapting it into whatever
+                solution is needed
```

Comment:
How about something like:

```suggestion
                Developed to encourage best practices, but flexible enough to stay out of your way
```

Reply:
That sounds better!